filter.txt

```diff
@@ -751,6 +752,7 @@ zoomtech.ir###text-3
 ||afkarnews.ir/serve/*.jpg
 ||afkarnews.ir/servev2/*.gif
 ||afkarnews.ir/servev2/*.jpg
+||aftabir.com/*/adv_images/*$image
```

Comment:
We need to keep this. The cosmetic filter (line 134-137) only hides the block, yet the browser downloads the image.

Reply:
@farbod-s I believe it would be best to keep both, as if the image url (or the class name of the element) changes, we can rely on the other to remove the ad.

Also; I'm not sure if this assumption is correct, but wouldn't including both types of the rules result in the url filterings to take effect first? If this isn't so, would moving cosmetic filters to the bottom of the file resolve this?

Reply:
Thanks for the update. I cannot confirm if this behavior is consistent across different browsers or plugins (e.g. ABP). As far as I can remember from a few years back, ABP would download the image in such cases. However this might have changed. I would suggest keeping the rule just in case.

Reply:
@farrokhi I will do some tests using ABP, but in any case I suggest merging this PR, as it is, for now.